acceptance/.gitignore

```diff
@@ -1,3 +1,5 @@
+# Do not add environment-specific entries here (see the top-level .gitignore
+# for reasoning and alternatives).
```

Comment:
The top level .gitignore doesn't have the reasoning for this change, just how to work around it. Is having extra ignores an issue?